src/registrar/tests/test_admin.py

```diff
@@ -886,6 +888,86 @@ def test_get_filters(self):
         self.assertContains(response, retrieved_html, count=1)
 
 
+class UserDomainRoleAdminTest(TestCase):
+    def setUp(self):
+        """Setup environment for a mock admin user"""
+        self.site = AdminSite()
+        self.factory = RequestFactory()
+        self.admin = ListHeaderAdmin(model=UserDomainRoleAdmin, admin_site=None)
+        self.client = Client(HTTP_HOST="localhost:8080")
+        self.superuser = create_superuser()
+
+    def tearDown(self):
+        """Delete all Users, Domains, and UserDomainRoles"""
+        User.objects.all().delete()
+        Domain.objects.all().delete()
+        UserDomainRole.objects.all().delete()
+
+    def test_email_not_in_search(self):
+        """Tests the search bar in Django Admin for UserDomainRoleAdmin.
+        Should return no results for an invalid email."""
+        # Have to get creative to get past linter
+        p = "adminpass"
+        self.client.login(username="superuser", password=p)
+
+        fake_user = User.objects.create(
+            username="dummyuser", first_name="Stewart", last_name="Jones", email="AntarticPolarBears@cold.com"
```

Comment:
**Comment (nonblocking):** Just bc I'm fussy can we fix the spelling to `antarctic` for this and all the other instances in this test LOL

Reply:
Fair enough, nice catch